README.md

```diff
@@ -60,6 +64,12 @@ You can update the catalog at anytime with the following command :
 
 *See Gunicorn documentation for more options*
 
+## Search for tools using the webapp
+Tools can be searched by:
+  * Key one or many words. Example: samtools
+  * EDAM ontology topics (see available topics in the Topics tab). Example: topics:conversion
+  * Instances to list all tools available on a given Galaxy instance. It requires from the instance to have a defined brand. Exemple: instance:galaxeast.
```

Comment:
instance: is a filter that can be added to other search keywords or filters
It is not just intended to list "all" tools available on a given instance.